bitcoin/src/crypto/key.rs

```diff
@@ -350,6 +350,12 @@ impl fmt::Display for CompressedPublicKey {
     }
 }
 
+impl fmt::Debug for CompressedPublicKey {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        f.write_fmt(format_args!("CompressedPublicKey({})", self))
```

Comment:
I wonder if we even need the surrounding `CompressedPublicKey()` - should be obvious it's compressed from the length and prefix. Anyway this is certainly an improvement.

Reply:
I believe it should be there. Otherwise it would look too close to any 32-byte hex-string of which there exist plenty.

Reply:
That's not an issue usually. Many types don't have the marker and that's fine.